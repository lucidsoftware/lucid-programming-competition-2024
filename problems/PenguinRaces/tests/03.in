15
Pingo - 18
Snowball - 26
Frosty - 24
Nibbles - 21
Marshmallow - 11
Shiver - 17
Frostbyte - 15
Speedy - 108
Pebbles - 23
Slider - 12
Goo - 13
Happy - 10
Honor - 22
Gus - 25
Willy - 42