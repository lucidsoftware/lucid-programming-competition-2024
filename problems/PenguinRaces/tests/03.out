First: Happy
Second: Marshmallow
Third: Slider